### PR TITLE
Change VBA grammar source

### PR DESCRIPTION
**Pull request by serkonda7**
Followup to #6677 
Remove the intermediary step of transpiling to json

## Checklist:

- [x] **I am changing the source of a syntax highlighting grammar**
  - Old: https://github.com/tiabeast/vscode-vba-json (does not exist anymore)
  - New: https://github.com/serkonda7/vscode-vba

### CODE REVIEW

File: vendor/licenses/git_submodule/vscode-vba.dep.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-name: vscode-vba-json
-version: cfe7e5671cce6e2529de0e68ca003cf96f3255f8
+name: vscode-vba
+version: 94e112416478e1f1728ce9ce88fc4ff1a9ed01ad
 type: git_submodule
-homepage: https://github.com/tiabeast/vscode-vba-json.git
-license: mpl-2.0
+homepage: https://github.com/serkonda7/vscode-vba.git
+license: other
```
Comment (lildude) [inline vendor/licenses/git_submodule/vscode-vba.dep.yml:6]:
```suggestion
license: mpl-2.0
```

Comment (serkonda7) [reply vendor/licenses/git_submodule/vscode-vba.dep.yml]: Thanks, missed that

**Review by lildude — APPROVED** [review]
LGTM. Thanks.

Note: this PR will not be merged until close to when the next release is made. See [here](https://github.com/github/linguist/blob/master/docs/troubleshooting.md#my-linguist-pr-has-been-merged-but-github-doesnt-reflect-my-changes) for more details.

**Comment by lildude** [comment]
Oooof, the missing repo is now causing other PRs to fail so I'm going to merge this now. It won't be used on GitHub.com until the next release.

**Comment by serkonda7** [comment]
Sorry for breaking your CI.

**Comment by cm9255176** [comment]
🍡

**Comment by cm9255176** [comment]
🥇 Meta

**Comment by cm9255176** [comment]
Zzzzz:

**Comment by cm9255176** [comment]
> Zzzzz:

**Pull request merged**